angular/projects/spark-angular/src/lib/components/sprk-stack/sprk-stack.stories.ts

```diff
@@ -19,7 +20,14 @@ export default {
     subcomponents: {
       SprkStackItemDirective,
     },
-    info: `${markdownDocumentationLinkBuilder('stack')}`,
+    info: `
+${markdownDocumentationLinkBuilder('stack')}
+- The \`itemSpacing\` @Input is not compatible with
+stack items with specific-column classes such
+as \`sprk-o-Stack__item--half@xs\`. Use padding
+instead. See the "Stack/Split - Two Column With Padding"
+for an example.
+`,
```

Comment:
indentation?
![tenor-16271520](https://user-images.githubusercontent.com/4342363/79390630-0d88f200-7f3e-11ea-8736-3b9ebffc4623.gif)

Reply:
@RVMendoza , This indentation is necessary as it is because its rendered as markdown and the white space will show in rendered text if we were to add it. So anytime we add "info" we have to remove white space.